nbm-maven-plugin/pom.xml

```diff
@@ -123,12 +123,6 @@ under the License.
         </dependency>
 
         <!-- BAD BOYS: we want to get rid of these -->
-        <dependency>
-            <groupId>org.apache.maven</groupId>
-            <artifactId>maven-compat</artifactId>
-            <version>${maven.version}</version>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
@cstamas was the comment only meant for `maven-compat` or for more deps in the list?

If its just for maven compat we have to remove the comment since it would be above the wrong lib now

Reply:
uh oh, this PR _adds it_? Big NO-NO

Reply:
It's handled in maven 4 ?